images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -472,4 +472,22 @@ function Get-SwiftLintVersion {
 function Get-PowershellVersion {
     $powershellVersion = Run-Command "powershell --version"
     return $powershellVersion
-}
\ No newline at end of file
+}
+
+function Build-PackageManagementEnvironmentTable {
+    return @(
+        @{
+            "Name" = "CONDA"
+            "Value" = $env:CONDA
+        },
+        @{
+            "Name" = "VCPKG_INSTALLATION_ROOT"
+            "Value" = $env:VCPKG_INSTALLATION_ROOT
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-PathWithLink($_.Value)
```

Comment:
These variables are not symlinks so we don't need to use `Get-PathWithLink` here